example-scripts/voctolight/testlight.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# Test driver for Voctolight plugins.
+from lib.config import Config
+from lib.plugins.all_plugins import PLUGINS
+
+def main():
+    plugin_cls = PLUGINS.get(Config.get('light', 'plugin'), None)
+    if plugin_cls is None:
+        print('No plugin selected, control will not work!')
```

Comment:
In this case there is no reason for this script to keep running, is it? It wil lwail on the next line anyway. So why not terminate with an Exception right here?

Reply:
Furthermore this code is replicated in `voctolight.py`. Better move it to `all_plugins.py` and have a clean interface `get_plugin_cls_by_name(name)` which either returns the `plugin_cls` or raises an Excaption.

Reply:
Done.